Orlando Walking Tours/Views/LocationTableViewCell.swift

```diff
@@ -7,11 +7,62 @@
 //
 
 import UIKit
+import Alamofire
 
-class LocationTableViewCell: UITableViewCell {
+class LocationTableViewCell: UITableViewCell, ReusableView
+{
+    ////////////////////////////////////////////////////////////
+    // MARK: - IBOutlets
+    ////////////////////////////////////////////////////////////
 
-    var locationId: String!
     @IBOutlet weak var locationThumbnail: UIImageView!
     @IBOutlet weak var locationTitle: UILabel!
     @IBOutlet weak var addLocationButton: UIButton!
+    @IBOutlet weak var loadingIndicator: UIActivityIndicatorView!
+
+    ////////////////////////////////////////////////////////////
+    // MARK: - Properties
+    ////////////////////////////////////////////////////////////
+
+    var locationId: String!
```

Comment:
I need to find out from @gregsbarr what this locationId is used for.  It may be something that can be removed completely.